runner.go

```diff
@@ -102,20 +129,69 @@ func generateUserSeed() error {
 	}
 }
 
+func appendFile(dst io.Writer, path string) error {
+    if src, err := os.OpenFile(path, os.O_RDONLY, 0644); err != nil {
+        log.Print("Failed to open file: ", err)
+        return err
+    } else {
+        defer src.Close()
+        if _, err := io.Copy(dst, src); err != nil {
+            log.Print("Failed to read file: ", err)
+            return err
+        }
+    }
+    _, err := dst.Write([]byte("\n"))
+    return err
+}
+
 func enableSplunkAPI() error {
+
 	if serverFile, err := os.OpenFile(os.ExpandEnv(ServerConfigPath), os.O_CREATE|os.O_TRUNC|os.O_WRONLY, 0644); err != nil {
 		return err
 	} else {
 		defer serverFile.Close()
-		_, err = fmt.Fprintf(serverFile, `[sslConfig]
-enableSplunkdSSL = false
-[httpServer]
-acceptFrom = 127.0.0.1/8
-[proxyConfig]
-http_proxy = %s
-https_proxy = %s
-no_proxy = %s
-`, os.Getenv("HTTP_PROXY"), os.Getenv("HTTPS_PROXY"), os.Getenv("no_proxy"))
+
+        if err := exec.Command("/usr/bin/update-ca-trust").Run(); err != nil {
+            log.Fatal(err)
+            return err
+        }
+
+        if caFile, err := os.OpenFile(os.ExpandEnv(SplunkTrustedBundle), os.O_CREATE|os.O_TRUNC|os.O_WRONLY, 0644); err != nil {
+            log.Fatal(err)
+            return err
+        } else {
+            defer caFile.Close()
+            if err := appendFile(caFile, TrustedCABundle); err != nil {
+                return err
+            }
+            if err := appendFile(caFile, os.ExpandEnv(SplunkAppsCACert)); err != nil {
+                return err
+            }
+        }
+
+
+        proxyConfig := ""
+
+        if http_proxy, exists := os.LookupEnv("HTTP_PROXY"); exists {
+            proxyConfig += "http_proxy = " + http_proxy + "\n"
+        }
+
+        if https_proxy, exists := os.LookupEnv("HTTPS_PROXY"); exists {
+            proxyConfig += "https_proxy = " + https_proxy + "\n"
+        }
+
```

Comment:
Do we need to catch any case where customer sets http_proxy or https_proxy but not the other, or is that config not supported?

Reply:
https_proxy is the only meaningful option currently, the HEC endpoint in splunk cloud is https-only

We can work around that if need be